pkg/apis/submariner.io/v1/types.go

```diff
@@ -81,6 +84,17 @@ type GatewayStatus struct {
 	Connections   []Connection `json:"connections"`
 }
 
+// LatencySpec describes the round trip time for a packet
+// between the gateway pod of different clusters this cluster
+// is connected.
```

Comment:
I assume the units are in nanosec.

```suggestion
// LatencySpec describes the round trip time information in nanoseconds for a packet
// between the gateway pods of two clusters.
```

Reply:
yes it is in nano-seconds